src/realm/object-store/sync/app.cpp

```diff
@@ -1067,8 +1068,8 @@ void App::call_function(const std::shared_ptr<SyncUser>& user, const std::string
         [self = shared_from_this(), name = name, service_name = std::move(service_name),
          completion = std::move(completion)](const Response& response) {
             if (auto error = AppUtils::check_for_errors(response)) {
-                self->log("App: call_function: %1 service_name: %2 -> %3 ERROR: %4", name, service_name,
-                          response.http_status_code, error->message);
+                self->log_debug("App: call_function: %1 service_name: %2 -> %3 ERROR: %4", name, service_name,
```

Comment:
I see this was logged at debug level before, but should it be an error instead?

Reply:
Yes, thank you